lib/email_parser/parser.rb

```diff
@@ -324,13 +324,31 @@ def self.strip_html(body, headers)
       return nil if body.nil?
 
       doc = Nokogiri::HTML(body)
-      # TODO: Handle non-gmail html sources
-      if headers.keys.include?('X-Google-Smtp-Source')
-        quoted_content = doc.xpath("//div[@class='gmail_quote']").first
-        quoted_content&.remove
 
-        signature_content = doc.xpath("//div[@class='gmail_signature']").first
-        signature_content&.remove
+      # remove quoted content
+      remove_element_onwards(doc.xpath("//div[@class='gmail_quote']").first) # gmail clients
+      remove_element_onwards(doc.xpath("//div[@id='appendonsend']").first) # office365 clients
+      remove_element_onwards(doc.xpath("//blockquote").first) ## apple clients
```

Comment:
Could we add superhuman as well? Or add a comment explaining it uses the same format as client X.

Reply:
great suggestion

Reply:
Confirmed that the superhuman uses the same html structure as gmail (same class names etc.)